src/psyclone/tests/parse/kernel_test.py

```diff
@@ -133,15 +180,59 @@ def test_getkernelfilepath_caseinsensitive2(tmpdir):
     assert "tmp" in result
     assert "test_mod.f90" in result
 
-# function get_kernel_metadata
+
+# function get_kernel_interface
+def test_get_kernel_interface_match():
+    ''' Tests that something is returned when searching for an interface'''
```

Comment:
Please extend: "...for an interface block in a parse tree that contains one."

Reply:
Done.